rotkehlchen/tests/exchanges/test_woo.py

```diff
@@ -0,0 +1,266 @@
+                    'vault': 0,
+                    'interest': 0,
+                    'pendingShortQty': 0,
+                    'pendingLongQty': 0,
+                    'availableBalance': 0,
+                    'averageOpenPrice': 0.23432,
+                    'markPrice': 0.25177,
+                    'updatedTime': 31321.121,
+                },
+                {
+                    'token': 'BTC',
+                    'holding': 0,
+                    'frozen': 0,
+                    'staked': 0,
+                    'unbonding': 0,
+                    'vault': 0,
+                    'interest': 0,
+                    'pendingShortQty': 0,
+                    'pendingLongQty': 0,
+                    'availableBalance': 0,
+                    'averageOpenPrice': 0.23432,
+                    'markPrice': 0.25177,
+                    'updatedTime': 31321.121,
+                },
+            ],
+        },
+    }
+
+    asset_balances = mock_woo._deserialize_accounts_balances(balances_response)
+    assert asset_balances == {
+        A_WOO: Balance(
+            amount=FVal('1'),
+            usd_value=FVal('1.5'),
+        ),
+        A_ETH: Balance(
+            amount=FVal('2'),
+            usd_value=FVal('3'),
+        ),
+    }
+
+
+def test_deserialize_trade_buy(mock_woo):
+    mock_trades = {
+        'id': 1,
+        'symbol': 'SPOT_BTC_ETH',
+        'order_id': 101,
+        'executed_price': 50000.0,
+        'executed_quantity': 1.0,
+        'side': 'BUY',
+        'fee': 0.1,
+        'fee_asset': 'ETH',
+        'executed_timestamp': '1634600000.0',
+    }
+    result = mock_woo._deserialize_trade(mock_trades)
+    assert result == Trade(
+        timestamp=1634600000,
+        location=Location.WOO,
+        base_asset=A_BTC,
+        quote_asset=A_ETH,
+        trade_type=TradeType.BUY,
+        amount=FVal('1'),
+        rate=FVal('50000'),
+        fee=FVal('0.1'),
+        fee_currency=A_ETH,
+        link='1',
+    )
+
+
+def test_deserialize_trade_sell(mock_woo):
+    mock_trades = {
+        'id': 2,
+        'symbol': 'SPOT_ETH_USDT',
+        'order_id': 102,
+        'executed_price': 3000.0,
+        'executed_quantity': 2.0,
+        'side': 'SELL',
+        'fee': 0.2,
+        'fee_asset': 'USDT',
+        'executed_timestamp': '1634610000.0',
+    }
+    result = mock_woo._deserialize_trade(mock_trades)
+    assert result == Trade(
+        timestamp=1634610000,
+        location=Location.WOO,
+        base_asset=A_ETH,
+        quote_asset=A_USDT,
+        trade_type=TradeType.SELL,
+        amount=FVal('2'),
+        rate=FVal('3000'),
+        fee=FVal('0.2'),
+        fee_currency=A_USDT,
+        link='2',
+    )
+
+
+def test_deserialize_asset_movement_deposit(mock_woo):
+    mock_deposit = {
+        'created_time': '1579399877.041',
+        'updated_time': '1579399877.041',
+        'id': '202029292829292',
+        'external_id': '202029292829292',
+        'application_id': 'null',
```

Comment:
is this string `'null'` or just `null`? if it is just `null` in python should be `None`